bundles/org.openhab.binding.comfoair/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -82,6 +118,23 @@
 		</channels>
 	</channel-group-type>
 
+	<channel-group-type id="ventilation_WHR930">
```

Comment:
If I see correctly, this is identical to `ventilation`. You could simply re-use the channel group type instead of defining it again.

Reply:
There is no `fanIn`/`Out3` for the WHR930, so not identical to `ventilation`. I tried to re-use channel groups were appropriate.

Reply:
I overlooked that fine difference.